packages/interface/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+### Breaking Changes
+
+-   `InterfaceSkeleton` no longer supports region navigation and its props `enableRegionNavigation` and `shortcuts` are removed. ([#63611](https://github.com/WordPress/gutenberg/pull/63611)). It’s recommended to add region navigation with the higher-order component `navigateRegions` or the hook `__unstableUseNavigateRegions` from `@wordpress/components`.
+
+## 6.9.0 (2024-10-03)
+
```

Comment:
@gziolo We need to remove this right as we only include patch releases in 6.7 tag?

Reply:
Yes, correct. The current version is `6.8.5` which is lower and it doesn't contain all the same changes as `6.9.0` published from `wp/latest` had at that time.

Reply:
```suggestion
```

Remove